core/src/avm2/globals/flash/display/DisplayObject.as

```diff
@@ -127,6 +130,18 @@ package flash.display {
 
         public native function globalToLocal(point:Point):Point;
 
+        [API("662")]
```

Comment:
Not directly related, but just noticed that `blendShader` has the same `[Version("10")]` metadata in `playerglobal.swc` as these functions (which should indeed translate to `[API("662")]`), but it's missing for us.